spring-cloud-kubernetes-config/src/test/java/org/springframework/cloud/kubernetes/config/MultipleConfigMapsTests.java

```diff
@@ -67,27 +67,23 @@ public static void setUpBeforeClass() {
 		System.setProperty(Config.KUBERNETES_NAMESPACE_SYSTEM_PROPERTY, "test");
 		System.setProperty(Config.KUBERNETES_HTTP2_DISABLE, "true");
 
-		createConfigmap(server, "s1", "defnamespace", new HashMap<String, String>() {
-			{
-				put("bean.common-message", "c1");
-				put("bean.message1", "m1");
-			}
-		});
-
-		createConfigmap(server, "defname", "s2", new HashMap<String, String>() {
-			{
-				put("bean.common-message", "c2");
-				put("bean.message2", "m2");
-			}
-		});
-
-		createConfigmap(server, "othername", "othernamespace",
-				new HashMap<String, String>() {
```

Comment:
there is nothing wrong with this anti-pattern, especially in tests. The biggest problem is the number of users that look at Spring code and might think - "hey, Spring does it!". As such, let's get rid of this common anti-pattern.